plugins/modules/system/parted.py

```diff
@@ -610,8 +618,9 @@ def main():
 
         # Create partition if required
         if part_type and not part_exists(current_parts, 'num', number):
-            script += "mkpart %s %s %s " % (
+            script += "mkpart %s %s %s %s " % (
                 part_type,
+                fs_type,
                 part_start,
                 part_end
             )
```

Comment:
```suggestion
            if fs_type is not None:
                script += "mkpart %s %s %s %s " % (
                    part_type,
                    fs_type,
                    part_start,
                    part_end
                )            
            else:
                script += "mkpart %s %s %s " % (
                    part_type,
                    part_start,
                    part_end
                )
```
If it isn't passed explicitly, it will be None